src/jvmMain/kotlin/ru/hse/sd/rogue/game/controller/GlobalController.kt

```diff
@@ -1,8 +1,36 @@
 package ru.hse.sd.rogue.game.controller
 
+import ru.hse.sd.rogue.game.logic.action.ActionsManager
+import ru.hse.sd.rogue.game.logic.action.registerRepeatable
+import ru.hse.sd.rogue.game.state.character.MobState
+import ru.hse.sd.rogue.game.state.character.PlayerState
+
 /**
  * Controls the whole game and checks whether it is finished or not.
  */
-class GlobalController : Controller {
-    // TODO
-}
\ No newline at end of file
+class GlobalController(
+    /**
+     * Actions manager to manage controlling of game.
+     */
+    actionsManager: ActionsManager,
+    /**
```

Comment:
```suggestion
    actionsManager: ActionsManager,
    
    /**
```
И ниже, обычно члены класса пустыми строками разделяют, чтобы они не сливались